packages/cozy-sharing/src/components/ShareByLink.jsx

```diff
@@ -168,6 +168,7 @@ class ShareByLink extends React.Component {
                     placement="bottom-end"
                     containerElRef={this.containerRef}
                     anchorElRef={this.containerRef}
+                    popperOptions={popperOptions ? popperOptions : undefined}
```

Comment:
```suggestion
                    popperOptions={popperOptions}
```

Is there really a need for ternary condition ?